src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj

```diff
@@ -53,7 +53,7 @@ This package is an internal implementation of the .NET Core SDK and is not meant
     <!-- Platform manifest and package override metatdata -->
     <ReferencePackSharedFxVersion>$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion).0</ReferencePackSharedFxVersion>
     <ReferencePackSharedFxVersion Condition="'$(VersionSuffix)' != ''">$(ReferencePackSharedFxVersion)-$(VersionSuffix)</ReferencePackSharedFxVersion>
-    <ReferencePlatformManifestOutputPath>$(ArtifactsObjDir)ref\PlatformManifest.txt</ReferencePlatformManifestOutputPath>
+    <ReferencePlatformManifestPath>$(RepoRoot)eng\PlatformManifest.txt</ReferencePlatformManifestPath>
```

Comment:
In non-servicing builds, this should copy from the just-built runtime pack. That way, we can merge this change into 'master'.